### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by magnusp**
- Remove docker compose capabilities
- Add a "prerequisites" section to the readme
- Dockerfile targets java 11
- Add .editorconfig
- Maven dependencymanagement and module naming

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,142 +29,180 @@
...
+    </dependencyManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                    <compilerArgument>-parameters</compilerArgument>
```
Comment (jepp3) [inline pom.xml]: Is it possible to have 11 here? So that we get the performance benefits from version 11?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ A user can add a question to rocket fuel directly in the ui, and then wait for a
 
 Feel free to contribute to rocket fuel. Create a issue, when the issue is accepted, create a fork and create a pull request. 
 
+#### Prerequisites
+
+* Authentication requires you to set up a client in the Google Developer console.
```
Comment (splitfeed) [inline README.md]: Are there any particular settings in the devconsole you need to set, or is it just "Create auth things" and it will work?

Comment (magnusp) [reply README.md]: Fixed!

File: pom.xml (hunk 1 above)
Comment (magnusp) [reply pom.xml]: Fixed!

**Pull request merged**